server/handler/boltdb/handler.go

```diff
@@ -207,8 +207,15 @@ func (h *boltdbHandler) Search(boundDN string, req *ldap.SearchRequest, conn net
 
 	entries, _ := h.bdb.Search(req.BaseDN, req.Scope)
 
+	// strip "userPassword" Attribute from the Results
+	var sr []*ldap.Entry
+	for _, e := range entries {
+		entry := ldapentry.EntryDropAttribute(e, "userPassword")
```

Comment:
Why is this only done for the bolt handler? Wouldn't it be better to deal with this on the database level? I don't remember how I dealt with this in the Ldif implementation.

Reply:
https://github.com/libregraph/idm/blob/8d51161be79569f00dd239ca3f8c6ad9b840ee44/server/handler/ldif/entry.go#L19 ah here - it's stored separately so its not returned, no extra computation required. I would prefer so see something similar for the bolt backend so it does not need any computation in the hot request path.